### PR TITLE
Create tester-20230801.md

### PR DESCRIPTION
**Pull request by itsnotalisher**

### CODE REVIEW

**Comment by itsnotalisher** [comment]
@dr-dimitru please  approve.

File: tester-20230801.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# QA/QC Tester
+
+## Description
+We are looking for a diligent and experienced Junior to mid-level QA/QC Tester to join our team. This role involves ensuring the quality of our Application through manual testing, script creation, and effective task management. The candidate will be pivotal in overseeing project life cycles, mitigating risks, managing issues, and aligning project documentation with developed features.
+
+## Responsibilities
+- Lead the manual testing process for our PWA, ensuring the software meets the highest quality standards.
```
Comment (dr-dimitru) [inline tester-20230801.md]: I won't attach to PWA, let's say it's web-stack, websites, and PWAs

File: tester-20230801.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- Create, manage, and execute test scripts to validate software functionality and user experience.
```
Comment (dr-dimitru) [inline tester-20230801.md]: ... test scripts (manual and automated)...

File: tester-20230801.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- Oversee task management, including task creation, assignment, and tracking.
+- Identify, report, and resolve project risks and issues.
+- Engage in project planning and design reviews, ensuring alignment with project objectives.
+- Facilitate effective communication within the team through daily scrum, sprint planning, and retrospective meetings.
+- Review project documentation to ensure consistency with developed features.
+
+## Deliverables
+- Execute rigorous manual testing of the PWA and create detailed test scripts.
```
Comment (dr-dimitru) [inline tester-20230801.md]: PWA => web-stack, websites, and PWAs

File: tester-20230801.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+- Manage task creation, assignment, and progress tracking.
+- Promptly identify and resolve project issues.
+- Deliver high-quality software within agreed deadlines.
+- Foster excellent team communication and collaboration.
+- Ensure project documentation accurately reflects the developed PWA features.
```
Comment (dr-dimitru) [inline tester-20230801.md]: PWA => web-stack, websites, and PWAs

**Review by dr-dimitru — CHANGES_REQUESTED** [review]
@itsnotalisher please see my comments for minor improvements

**Comment by itsnotalisher** [comment]
@dr-dimitru changes made, please commit

**Review by dr-dimitru — APPROVED** [review]

**Pull request merged**